packages/shorebird_cli/lib/src/artifact_manager.dart

```diff
@@ -285,4 +285,25 @@ Failed to create diff (exit code ${result.exitCode}).
       ),
     );
   }
+
+  /// Returns the .app directory generated by `flutter build ipa`. This was
+  /// traditionally named `Runner.app`, but can now be renamed.
+  Directory? getAppDirectory({required Directory xcarchiveDirectory}) {
```

Comment:
Maybe `getIosAppDirectory` for clarity?

Reply:
Just realized that that API already exists, closing this 👍